lib/Versions/VersionsBackend.php

```diff
@@ -151,12 +151,12 @@ public function getVersionFile(IUser $user, FileInfo $sourceFile, $revision): Fi
 	}
 
 	/**
-	 * @param array $folder
+	 * @param array{id: int, mount_point: mixed, groups: array<empty, empty>|mixed, quota: mixed, size: int, acl: bool} $folder
 	 * @return (FileInfo|null)[] [$fileId => FileInfo|null]
 	 */
 	public function getAllVersionedFiles(array $folder) {
 		$versionsFolder = $this->getVersionsFolder($folder['id']);
-		$mount = $this->mountProvider->getMount($folder['id'], '/dummyuser/files/' . $folder['mount_point'], $folder['groups'], $folder['quota']);
```

Comment:
Can \OCP\Constant::PERMISSION_ALL be used instead of 31 here?

Reply:
Good idea